pkg/healthautoexport/types.go

```diff
@@ -29,10 +29,11 @@ type PayloadData struct {
 
 // Metric defines a single measurement with units, as well as time-series data points.
 type Metric struct {
-	Name          string           `json:"name"`
-	Units         Units            `json:"units"`
-	Datapoints    []*Datapoint     `json:"-"`
-	SleepAnalyses []*SleepAnalysis `json:"-"`
+	Name                   string                    `json:"name"`
+	Units                  Units                     `json:"units"`
+	Datapoints             []*Datapoint              `json:"-"`
+	SleepAnalyses          []*SleepAnalysis          `json:"-"`
+	AggregateSleepAnalyses []*AggregateSleepAnalysis `json:"-"`
```

Comment:
nit: Maybe we can name it `AggregatedSleepAnalyses` (and the type be `AggregatedSleepAnalysis`)

Reply:
Sounds good to me.